src/arraySchema.ts

```diff
@@ -0,0 +1,60 @@
+import { Dictionary, expandUrl, loadField, typeDSLenumd062602be0b4b8fd33e69e29a841317b6ab665bcLoader2, UriUnionOfPrimitiveTypeLoaderOrRecordSchemaLoaderOrEnumSchemaLoaderOrArraySchemaLoaderOrStrtypeOrArrayOfUnionOfPrimitiveTypeLoaderOrRecordSchemaLoaderOrEnumSchemaLoaderOrArraySchemaLoaderOrStrtypeFalseTrue2, LoadingOptions, Saveable, ValidationException } from './util/internal'
```

Comment:
This is hand-written? :-)

Reply:
I coped it from the python generated code and changed the casing 😄